resources/aws/ec2/vm.go

```diff
@@ -21,12 +21,20 @@ type InstanceArgs struct {
 	InstanceProfile string
 
 	// Optional
-	UserData string
+	UserData           string
+	HTTPTokensRequired bool
```

Comment:
By default this is false, but I think it makes sense to have it enabled by default since it's the case on AWS (or it will be very soon)

Reply:
https://aws.amazon.com/fr/blogs/aws/amazon-ec2-instance-metadata-service-imdsv2-by-default/
> Mid-2024 – Newly released Amazon EC2 instance types will use IMDSv2 only by default.